### PR TITLE
NAS-121664 / 23.10 / Stop or start only the necessary applications

### PR DESCRIPTION
**Pull request by bvasilenko**
**Testing**

Please follow repro steps described in the ticket, and make sure actual behavior now matches the expected one

### CODE REVIEW

**Review by denysbutenko — CHANGES_REQUESTED** [review]
It also uses different terms, `checkedAppsNames()` vs `startedSelectedApps()`. So let's use consistent naming.

File: src/app/pages/apps/components/installed-apps/installed-apps.component.ts [hunk 1]
```diff
@@ -115,6 +109,14 @@ export class InstalledAppsComponent implements OnInit, AfterViewInit {
       .some((app) => app.update_available || app.container_images_update_available);
   }
 
+  get startedSelectedApps(): ChartRelease[] {
+    return this.dataSource.filter((app) => app.status !== ChartReleaseStatus.Stopped && app.selected);
```
[the remark this answers is not in the document]
Comment (bvasilenko) [reply src/app/pages/apps/components/installed-apps/installed-apps.component.ts:113]: It's not very clear here: checkedAppsNames - returns a list of names (strings) of selected applications. And startedSelectedApps returns an array of started applications (ChartRelease[]). Is renaming really necessary?

@denysbutenko would you please clarify?

[the remark this answers is not in the document]
Comment (denysbutenko) [reply src/app/pages/apps/components/installed-apps/installed-apps.component.ts:113]: @bvasilenko Just wanted to give you a heads-up that some methods use the word `checked` while others use `selected` which might be a bit confusing.

**Review by undsoft — APPROVED** [review]
Looks good.

**Pull request merged**